powerquery-docs/data-privacy-firewall.md

```diff
@@ -3,15 +3,12 @@ title: Behind the scenes of the Data Privacy Firewall
 description: Describes the purpose of the Data Privacy Firewall
 author: ehrenMSFT
 ms.topic: conceptual
-ms.date: 1/9/2023
+ms.date: 8/31/2023
 ms.author: ehvonleh
 ---
 
 # Behind the scenes of the Data Privacy Firewall
 
->[!NOTE]
->Privacy levels are currently unavailable in Power Platform dataflows. The product team is working towards re-enabling this functionality in the coming weeks.
```

Comment:
From what I can tell, Dataflows in Power Apps still don't support privacy levels. Are you seeing something different?